src/index.js

```diff
@@ -0,0 +1,42 @@
+import './styles/main.scss';
+
+const userInput = document.querySelector('.task-input input');
+let tasks = JSON.parse(localStorage.getItem('Tasks'));
```

Comment:
- [ ] For this milestone, you don't need to get data from the `localStorage`. Your array will be static for now. Kindly implement it. 👍